cornflow/tests/unit/test_dags.py

```diff
@@ -23,14 +26,30 @@ def test_manual_dag(self):
         )
         payload_to_send = {**self.payload, **data}
         token = self.create_service_user()
-        # idx = self.create_new_row(
-        #     url=DAG_URL,
-        #     model=self.model,
-        #     payload=payload_to_send,
-        #     check_payload=False,
-        #     token=token,
-        # )
-        # print(idx)
+        idx = self.create_new_row(
+            url=DAG_URL,
+            model=self.model,
+            payload=payload_to_send,
+            check_payload=False,
```

Comment:
dumb question: could we be able to validate with `check_payload=True`? I had it at `False` in the commented code but I'm not sure if it was because we could not check or because I was lazy.

Reply:
We could, but we have to check which keys we are going to compare, as the sent data and the response data do not have the same keys. Let me check if I can change it